python/deltalake/table.py

```diff
@@ -1430,7 +1432,9 @@ def repair(
             commit_properties,
             post_commithook_properties,
         )
-        return json.loads(metrics)
+        deserialized = json.loads(metrics)
+        deserialized[FSCK_METRICS_FILES_REMOVED_LABEL] = json.loads(deserialized["files_removed"])
+        return deserialized
```

Comment:
if we don't care about maintaining the current response then we can omit this step, but for sake of breaking the interface, added a deserializing step here.